lib/standalone.js

```diff
@@ -164,6 +164,9 @@ async function standalone({
 		fileList = fileList.concat(ALWAYS_IGNORED_GLOBS.map((glob) => `!${glob}`));
 	}
 
+	// do not cache if config is explicitly overrided by option
+	const useCache = cache || (config?.cache && cache !== false);
```

Comment:
[suggest] I have the same suggestion as above:

```suggestion
	const useCache = cache ?? config?.cache ?? false;
```